espresso-support/dependencies.gradle

```diff
@@ -1,23 +1,25 @@
 ext {
-    androidCompileSdkVersion = 25
-    androidBuildToolsVersion = '25.0.1'
-    androidMinSdkVersion = 16
-    androidTargetSdkVersion = 25
-
-    supportLibraryVersion = '23.1.1'
-
-    librariesBuild = [
-        androidGradleBuildTools: 'com.android.tools.build:gradle:2.3.0',
-        bintrayRelease         : 'com.novoda:bintray-release:0.4.0'
+    versions = [
+            androidSdk    : [
+                    compile   : 25,
+                    min       : 16,
+                    target    : 25,
+                    buildTools: '25.0.1'
```

Comment:
commit 9425c9a537cb71d8239eb0e1ff392ce7438ecf13
Author: Ataul Munim <hello@ataulm.com>
Date:   Fri Mar 10 16:08:14 2017 +0000

    Update android build tools and support library version